contents.txt

```diff
@@ -2202,6 +1794,7 @@
 ./sdk/6.0.100/Sdks/FSharp.NET.Sdk/Sdk/Sdk.props
 ./sdk/6.0.100/Sdks/FSharp.NET.Sdk/Sdk/Sdk.targets
 ./sdk/6.0.100/Sdks/Microsoft.Docker.Sdk/
+./sdk/6.0.100/Sdks/Microsoft.Docker.Sdk/microsoft.docker.sdk.1.1.0.csproj
```

Comment:
> [@omajid]
> This file seems like something didn't get filtered out. Could this break any users? It just won't get used, right?

Reply:
I don't think `.csproj` would be automatically picked up by anything, so I think it's safe. This issue tracks fixing this kind of file being included from SBRP:
* https://github.com/dotnet/source-build/issues/1906

It's currently in dotnet-sdk-5.0 in Fedora 34:
```
/usr/lib64/dotnet/sdk/5.0.206/Sdks/Microsoft.Docker.Sdk/Microsoft.Docker.Sdk.csproj
```